test/transform.jl

```diff
@@ -38,7 +38,13 @@ function single_sample_tests(dist)
 
     # Check that invlink is inverse of link.
     x = rand(dist)
-    @test @inferred(invlink(dist, link(dist, copy(x)))) ≈ x atol=1e-9
+
+    if dist isa LKJCholesky
+        x_inv = @inferred Union{Cholesky{Float64, Matrix{Float64}}, Matrix{Float64}} invlink(dist, link(dist, copy(x)))
```

Comment:
Shouldn't this be concrete now rather than `Union`? I.e.
```suggestion
        x_inv = @inferred invlink(dist, link(dist, copy(x)))
```

Reply:
Yes, good catch! I missed this one.